[2025-10-03T00:13:05.473+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-10-03T00:13:05.524+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: first_taskk.task_prnt scheduled__2025-10-03T00:12:00+00:00 [queued]>
[2025-10-03T00:13:05.539+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: first_taskk.task_prnt scheduled__2025-10-03T00:12:00+00:00 [queued]>
[2025-10-03T00:13:05.540+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-10-03T00:13:05.571+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): task_prnt> on 2025-10-03 00:12:00+00:00
[2025-10-03T00:13:05.591+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=851) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-10-03T00:13:05.595+0000] {standard_task_runner.py:63} INFO - Started process 853 to run task
[2025-10-03T00:13:05.595+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'first_taskk', 'task_prnt', 'scheduled__2025-10-03T00:12:00+00:00', '--job-id', '231', '--raw', '--subdir', 'DAGS_FOLDER/dag1.py', '--cfg-path', '/tmp/tmp368bny7b']
[2025-10-03T00:13:05.602+0000] {standard_task_runner.py:91} INFO - Job 231: Subtask task_prnt
[2025-10-03T00:13:05.628+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:195: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2025-10-03T00:13:05.728+0000] {task_command.py:426} INFO - Running <TaskInstance: first_taskk.task_prnt scheduled__2025-10-03T00:12:00+00:00 [running]> on host 452c0657f430
[2025-10-03T00:13:06.025+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='first_taskk' AIRFLOW_CTX_TASK_ID='task_prnt' AIRFLOW_CTX_EXECUTION_DATE='2025-10-03T00:12:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-10-03T00:12:00+00:00'
[2025-10-03T00:13:06.038+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-10-03T00:13:06.045+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-10-03T00:13:06.059+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'date']
[2025-10-03T00:13:06.123+0000] {subprocess.py:86} INFO - Output:
[2025-10-03T00:13:06.154+0000] {subprocess.py:93} INFO - Fri Oct  3 00:13:06 UTC 2025
[2025-10-03T00:13:06.156+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-10-03T00:13:06.160+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-10-03T00:13:06.251+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=first_taskk, task_id=task_prnt, run_id=scheduled__2025-10-03T00:12:00+00:00, execution_date=20251003T001200, start_date=20251003T001305, end_date=20251003T001306
[2025-10-03T00:13:06.336+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-10-03T00:13:06.398+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-10-03T00:13:06.403+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
